### ISSUE TITLE
Zip doesn't contain dataset

### ISSUE THREAD

**Issue by mlamelas** [codelab-tf-got/codelab-tf-got.github.io#1]
When downloading the zip file, dataset folder is empty. I had to download directly from https://github.com/codelab-tf-got/gameofthrones/archive/59457156fce22c8048b740a7b7a0ebf32f329b15.zip

**Comment by ssaavedra** [comment]
Yes, GitHub seems to have changed their behaviour. Previously, zip files included git submodules, but it seems it does not anymore, so it seems we will have to manually upload a zip file containing both things as a release.

As a workaround, you can manually download the dataset archive, or clone both repos simultaneously, as:

`git clone --recursive https://github.com/codelab-tf-got/code.git`
